flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/debezium/DebeziumUtils.java

```diff
@@ -233,4 +235,79 @@ private static Map<String, String> querySystemVariables(
 
         return variables;
     }
+
+    public static BinlogOffset findBinlogOffset(long targetMs, MySqlConnection connection) {
+        MySqlConnection.MySqlConnectionConfiguration config = connection.connectionConfig();
+        BinaryLogClient client =
```

Comment:
BinaryLogClient should be closed after finding the target offset.

Reply:
@YesOrNo828 it has been closed in Listener at last commit.